pytorch/training/docker/1.12/py3/Dockerfile.cpu

```diff
@@ -129,8 +129,7 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
     # Below 2 are included in miniconda base, but not mamba so need to install
     conda-content-trust \
     charset-normalizer \
-    # TMP FIX, NEED TO FIX SMDEBUG NUMPY DEPRECATION ERROR SOON: https://github.com/awslabs/sagemaker-debugger/blob/ff3b53607c36008164372b4fa785545f2e3ec2ef/smdebug/core/hook.py#L737
-    numpy=1.23.5 \
+    "shap<1" \
```

Comment:
are we removing numpy?

Reply:
yes we will be using the latest one from conda since smdebug is fixed